repo/packages/H/httpbin/0/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "HTTP Request & Response Service",
```

Comment:
Please add a link to the dcos/examples at the end of `description`. 
Something like: `Installation Documentation: https://github.com/dcos/examples/tree/master/1.8/httpbin`